bounties/level-2/video-tutorials.md

```diff
@@ -0,0 +1,55 @@
+# Video tutorials
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 750 XLM (+750 XLM for established creators) | 1000 XLM | Continuous 
+
+[//]: # (make sure to replace the file-name placeholders `<BOUNTY_FILE_NAME_NO_EXTENSION>`, `<BOUNTY_FILE_NAME_WITH_EXTENSION> and `<LEVEL>` in the next two lines with the respective values)
+📜&nbsp; View [existing submissions](https://github.com/tyvdh/stellar-quest-bounties/issues?q=is%3Aissue+label%3Avideo-tutorials) for this bounty. \
+🔵&nbsp; Start [hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.yml&link=https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/video-tutorials.md) this bounty.
+
+## Description
+
+### What is this task?
+
+Some developers love reading tutorials, some prefer watching videos to easily follow what to expect. Your task is to help increase recognition of Stellar ecosystem by producing a video walkthrough with (but not limited to) below topics:
+
+ - Explain like I'm Five: everything about Stellar ecosystem
+ - Implementation of Stellar Ecosystem Proposals (aka SEPs) in multiple coding languages
+ - Simple functional projects to demonstrate Stellar available operations
+
+
+### What are the requirements for the bounty hunter?
+
+ - The bounty hunter should have good knowledge of Stellar and can create productive walkthrough videos. They should be familiar with Stellar supported SDKs.
+ - `(Optional)` The bounty hunter could double the reward if they have a well-established viewer base such as:
+   - A Youtube channel with more than 1000 subscribers
+   - A tutorial blog with more than 10,000 pageviews per month
+
+### What are the deliverables?
+
+
+#### Bounty Description
+  - The hunter should create a minimum 2-min video
+
+#### Review Criteria
+  - The quality of content in the video
+  - The quality of the code being presented 
+    - Since people would probably copy and paste the code while watching, the code quality should be checked
+    - Should avoid less secure and performant source code, unless stated during the walkthrough
+  
+#### Submission Procedure
+  - Create a work issue 
+  - Add a link to your video in the work issue
+  - Upload the final video on Youtube and create a PR to upload it to assets folder
```

Comment:
I'd rather have some place where we collect the links to the videos. Maybe as an article on or tag-collection medium.com.

Reply:
Maybe Youtube video with tags? Or when we have a website for bounties, we can gather there.